src/Microsoft.Graph.Auth/Microsoft.Graph.Auth.csproj

```diff
@@ -28,7 +26,7 @@ New functionality
     <DelaySign>true</DelaySign>
     <!--Skip 1.13.0-->
     <VersionPrefix>0.1.0</VersionPrefix>
-    <VersionSuffix>preview</VersionSuffix>
+    <VersionSuffix>preview2</VersionSuffix>
```

Comment:
It should be preview.2  https://semver.org/#spec-item-9

Reply:
@darrelmiller Good catch. I have renamed as part of the latest commit.